pkgs/development/python-modules/sqlalchemy/default.nix

```diff
@@ -1,27 +1,22 @@
-{ stdenv, lib, fetchPypi, buildPythonPackage, isPy3k, isPy35, fetchpatch
+{ stdenv, lib, fetchPypi, buildPythonPackage, isPy27, isPy3k, pythonAtLeast
 , mock
+, greenlet
 , pysqlite
 , pytestCheckHook
 , pytest_xdist
 }:
 
 buildPythonPackage rec {
   pname = "SQLAlchemy";
-  version = "1.3.23";
+  version = "1.4.7";
+
+  disabled = !(isPy27 || pythonAtLeast "3.6");
```

Comment:
```suggestion
  disabled = pythonOlder "3.6";
```

Reply:
python-2.7 is still supported by sqlalchemy-1.4. Only versions before 3.6 in the 3 series are not (i.e between 2.7 excluded to 3.5 included).

Considering we do not ship any python3 release before 3.6 in nixpkgs, it looks like this all line could be dropped altogether. I admit having the requirement explicitly stated are good, and do not mind if they are kept.